packages/remix/src/utils.ts

```diff
@@ -29,16 +31,22 @@ export function assertValidClerkState(val: any): asserts val is ClerkState | und
  * @param name
  * @returns
  */
-export const getEnvVariable = (name: string): string => {
+export const getEnvVariable = (name: string, context?: AppLoadContext): string => {
   // Node envs
   if (typeof process !== 'undefined') {
     return (process.env && process.env[name]) || '';
   }
 
+  // Cloudflare pages
+  if (context) {
+    const contextEnv = context?.env as Record<string, string>;
```

Comment:
@octoper was this something that changed in v2, or we always had it wrong ?

Reply:
@panteliselef it's not that we were doing it wrong, but it's the preferred way on how to pass the environment variables to the context as it's show in the Remix Docs ([ref](https://remix.run/docs/en/main/guides/envvars#:~:text=getLoadContext%3A%20(context)%20%3D%3E%20(%7B%20env%3A%20context.env%20%7D)%2C)).